src/EnergyPlus/WeatherManager.cc

```diff
@@ -4324,12 +4324,14 @@ namespace WeatherManager {
                 flags.ADVANCE("No");
                 ObjexxFCL::gio::write(OutputFileInits, fmtA, flags) << StringOut;
             }
+
+            // Hum Ind Type, Hum Ind Value at Max Temp
             if (DesDayInput(EnvrnNum).HumIndType == DDHumIndType_WetBulb) {
                 StringOut = "Wetbulb," + RoundSigDigits(DesDayInput(EnvrnNum).HumIndValue, 2) + " {C},";
             } else if (DesDayInput(EnvrnNum).HumIndType == DDHumIndType_DewPoint) {
                 StringOut = "Dewpoint," + RoundSigDigits(DesDayInput(EnvrnNum).HumIndValue, 2) + " {C},";
             } else if (DesDayInput(EnvrnNum).HumIndType == DDHumIndType_Enthalpy) {
-                StringOut = "Enthalpy," + RoundSigDigits(DesDayInput(EnvrnNum).HumIndValue, 2) + " {kJ/kg},";
+                StringOut = "Enthalpy," + RoundSigDigits(DesDayInput(EnvrnNum).HumIndValue, 2) + " {J/kg},";
```

Comment:
Adjust unit. An alternative is to do `RoundSigDigits(DesDayInput(EnvrnNum).HumIndValue / 1000., 2) + " {kJ/kg}`

Reply:
Units here should be J/kgDryAir to be thorough.

Reply:
All of these should have a comma before the units per comments above - want to keep strings and numerics separate in this output.